manifests/charts/gateways/istio-egress/templates/poddisruptionbudget.yaml

```diff
@@ -1,5 +1,5 @@
-{{- if .Values.global.defaultPodDisruptionBudget.enabled }}
 {{ $gateway := index .Values "gateways" "istio-egressgateway" }}
+{{- if and $gateway.autoscaleEnabled .Values.global.defaultPodDisruptionBudget.enabled }}
```

Comment:
Should we check if pod disruption budget min available is less than hpa min replica before setting pdb?

Reply:
Or should we add some check into istioctl to make sure user does not make invalid config with pdb+hpa?